mastering-plone/behaviors_1.rst

```diff
@@ -3,11 +3,13 @@
 Behaviors
 =========
 
-.. sidebar:: Get the code!
+.. sidebar:: Get the code! (:doc:`More info <code>`)
 
-    Get the code for this chapter (:doc:`More info <code>`):
+   Code for the beginning of this chapter::
 
-    ..  code-block:: bash
```

Comment:
Please do not remove `code-block` nor trust the default language for syntax highlighting with `::`.

Please restore multiple instances of this.

Reply:
According to some docs the way Philip did it is also valid for simple code snippets : https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#how-to-include-simple-code
I will leave that up to Philip to decide if he wants to change it :-)

Reply:
No. The default syntax highlighting is Python, because there is no setting for `highlight_language` in the root `conf.py`. The code sample is bash, but gets highlighted as Python. This is suboptimal.

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#showing-code-examples

Reply:
I'm going to take your word for it, I'm in no means an rst expert. When I tested it in my local setup there was no visual difference between the two, so it seemed a valid option for this specific usecase:

![Screenshot 2019-11-08 10 53 00](https://user-images.githubusercontent.com/1117575/68467539-a8a2d080-0216-11ea-9b88-c64c9ec9a4f8.png)
I'm not going to fight over it ;-)

Reply:
Experience level in Sphinx is not required. Inspect the HTML markup that gets rendered. That is what matters. The name of the language appears as part of the markup and is what is used by Sphinx and Pygments to apply correct styling. It's coincidence that you see no visual difference.
![Google Chrome006](https://user-images.githubusercontent.com/102112/68496580-90778500-0207-11ea-9c95-2cc13ee78b1e.png)